### ISSUE TITLE
Opensuse 42.1 GNOME 13.6.2 two issues

### ISSUE THREAD

**Issue by aleksandrmelnyk** [raelgc/scudcloud#380]
1. When desktop notifications are on and scudcloud is not currently focused/active, eg. on background, and message received then whole GUI just hangs for several seconds, which is bit of annoying, because I cannot switch immediately back to scudcloud.
2. Stay logged in / remember me option not working

**Comment by raelgc** [comment]
@stieg Please: can you see if the first issue is reproducible for you? Which Gnome version are you using?

**Comment by stieg** [comment]
The first issue sounds more like a Gnome issue than a scudcloud issue.  I have seen this behavior before.  IIRC the cause is long GC times in the UI thread when messages come in.  But I can try and repro on my Gnome (which is at version 3.18.3).

**Comment by stieg** [comment]
Tried reproducing on Gnome 3.18.3 on Fedora 23 and was not able to see the issue.

@mamayoleksandr Does the issue always happen with scudcloud or does it happen with other services as well.  If the former, how do you know specifically that it is scudcloud?  If you are uncertain I suggest trying some other apps and having them send notifications to see if the issues occur with them as well.  This will allow us to determine of the cause is scudcloud or Gnome itself.

For issue 2 what are the permissions on the `$HOME/.config/scudcloud` directory?  Here are what mine look like:

``` sh
[stieg@foo ~]$ ls -la $HOME/.config/scudcloud
total 16
drwxrwxr-x 1 stieg stieg    52 Oct 18 22:14 .
drwxr-xr-x 1 stieg stieg  1104 Jan 26 14:06 ..
drwxrwxr-x 1 stieg stieg    32 Oct 18 22:14 data7
drwxrwxr-x 1 stieg stieg     0 Feb  8 08:49 prepared
-rw-rw-r-- 1 stieg stieg 14332 Feb  8 08:43 scudcloud.cfg
```

**Comment by aleksandrmelnyk** [comment]
Hello guys,
1. so, I have just tried to call gnome notification with command:
`notify-send -i /../resources/images/user-message.png -h string:sound-file:/usr/share/sounds/gnome/default/alerts/sonar.ogg  "SUMMARY" "BODY"`
and gnome DID NOT hang. But with scudcloud it hung like about for 8-10 seconds. Am not sure, may be it is gnome issue, but depends on notification parameters.
2. Can confirm it was permission issue. I have just tried chmod -R 0755 and it started to work

thanks,

Alex.
